src/components/schemas/fields.yaml

```diff
@@ -1302,3 +1302,27 @@ mileage_settings:
         "two_wheeler": 44
   description: |
     Mileage settings of the employee.
+
+employee_filter_name:
+  type: string
+  nullable: false
+  maxLength: 255
+  description: |
+    This represents the name of the filter.
+  example: Code E0101
+
+shared:
+  type: boolean
+  nullable: true
+  description: |
+    This indicates if the filter should be shared with other admins in the org.
+  default: false
+  example: true
+
+query_string:
+  type: string
+  nullable: false
+  maxLength: 255
+  description: |
+    This represents a query string which will be used to filter employees.
+  example: or=(and(code.eq.'E0101'))
```

Comment:
`or` part is not used, either add expression for `and` & `or`, else we can keep it simple, eg `code.eq.'E0101'`

Reply:
Got it, updated the docs
Thanks!